internal/config/config.go

```diff
@@ -38,6 +38,12 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
+// Set all the configurations for unit testing mode
```

Comment:
I'm not a fan of this comment.
It doesn't really add any information that the name of the function doesn't already have.